src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +121,22 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Parses and returns the module name.
+     */
+    public static String parseModuleName(String name) {
+        requireNonNull(name);
+        String trimmedName = name.trim();
+        return trimmedName;
+    }
+
+    /**
+     * Parses and returns the module code to serve as an identifier.
+     */
+    public static String parseModuleCode(String code) {
+        requireNonNull(code);
+        String trimmedCode = code.trim();
+        return trimmedCode;
+    }
```

Comment:
Do these methods not have is valid checks? In that case we could just implement a generic method.

eg. a parsePrefixValue()

If these are stubs then it's fine.